packages/fw-theme-builder/src/fw-theme-builder.ts

```diff
@@ -0,0 +1,313 @@
+    if (this.nav == "Colors" || this.nav == "Sizes" || this.nav == "Fonts") {
+      this.nav = "home";
+    } else {
+      this.nav = "Colors";
+    }
+  }
+
+  createFontPickComponent(font: string) {
+    return html` <fw-font-pick
+      exportparts="font-container, font-label, font-button, font-dropdown-container, font-dropdown-option, font-dropdown-selected"
+      .label=${font}
+      @value-changed=${(e: any) => {
+        let detail = {
+          type: font,
+          value: e.detail.value,
+        };
+        this.shadowRoot?.dispatchEvent(
+          new CustomEvent("font-change", {
+            detail,
+            bubbles: true,
+            composed: true,
+          })
+        );
+        this.theme.Fonts[font] = e.detail.value;
+        this.theme = { ...this.theme };
+      }}
+      .options="${this.fontOptions}"
+      .value=${this.theme.Fonts[font]}
+    >
+    </fw-font-pick>`;
+  }
+
+  createSizePickComponent(size: string) {
+    return html` <fw-size-pick
+      exportparts="size-container, size-label, size-input"
+      .label=${size}
+      @value-changed=${(e: any) => {
+        let detail = {
+          type: size,
+          value: e.detail.value,
+        };
+        this.shadowRoot?.dispatchEvent(
+          new CustomEvent("size-change", {
+            detail,
+            bubbles: true,
+            composed: true,
+          })
+        );
+        this.theme.Sizes[size] = e.detail.value;
+        this.theme = { ...this.theme };
+      }}
+      .value=${this.theme.Sizes[size]}
+    >
+    </fw-size-pick>`;
+  }
+
+  createColorPickComponent(group: string, type: string) {
+    return html` <fw-color-pick
+      exportparts="color-button, color-label, color-hidden-input"
+      @value-changed=${(e: any) => {
+        let detail = {
+          group: group,
+          type: type,
+          value: e.detail.hex,
+          rgb: e.detail.rgb,
+          hsl: e.detail.hsl,
+        };
+        this.shadowRoot?.dispatchEvent(
+          new CustomEvent("color-change", {
+            detail,
+            bubbles: true,
+            composed: true,
+          })
+        );
+        this.theme.Colors[group][type] = e.detail.hex;
+        this.theme = { ...this.theme };
+      }}
+      .label=${`${group} ${
+        type?.toLowerCase().slice(-3) == "hex"
+          ? type?.toLowerCase().slice(0, -4)
+          : type
+      }`}
+      .value="${this.theme.Colors[group][type]}"
+    >
+    </fw-color-pick>`;
+  }
+
+  getContent() {
+    let content;
+    switch (this.nav) {
+      case "home":
+        content = html` <div part="content-container">
+          ${(Object.keys(this.theme) ?? []).map(
+            (section) => html` <button
+              part="theme-button"
+              @click=${(e: any) => this.sectionChangeHandler(e, `${section}`)}
+            >
+              ${section}
+            </button>`
+          )}
+        </div>`;
+        break;
+      case "Colors":
```

Comment:
Better to keep these all uppercase or all lowercase. Even better to have them as enums.